### ISSUE TITLE
Add Partner Logos to the footer 

### ISSUE THREAD

**Issue by jessiofhall** [rapidpro/chpro-microsite#101]
From @scampo 

> So we are thinking of this: https://centre.humdata.org/
> As an example
> But would need two levels, e.g. "Technical Support / Lead" and then a UNICEF logo
> and then "Collaborators" and the other org logos
> 
> 1. Bring the About / Applications / Use Cases / Case Studies / Design for Impact MENU into vertical style (like on the OCHA site) on left hand side
> 2. Replace RapidPro HEALTH logo with UNICEF and then adjust text to be above the logo
> 3. Replicate same style of TEXT with logo(s) below for 'Collaborators" to the right of the UNICEF logo
> Make sense? @jessiofhall
> 4. we would have ability in CMS to (a) adjust the text above the two logo areas and (b) add new logos as needed
> 5. We remove "Terms" and "Privacy Policy" for now (just un-publish these so we could add them back in future as needed) and change "Support" to "Contact Us"

Next Steps: 

- Evaluate the time needed to accomplish this
- Determine if there's an alternate route worth considering/discussing 
- Determine an ETA

**Comment by sodevious** [comment]
@jessiofhall yes working on it! chatted through it with @mtrythall this morning

**Comment by jessiofhall** [comment]
@sodevious I don't have those at the moment. @scampo just needs to be able to add them via the CMS if that's possible. 

Is there a solution for that?

**Comment by jessiofhall** [comment]
![screen shot 2018-05-29 at 1 51 59 pm](https://user-images.githubusercontent.com/12751372/40678959-810c8828-6347-11e8-98b6-384bc13e541f.png)

That's an example he sent our way. I can dig for more info, though. :)

**Comment by sodevious** [comment]
@jessiofhall  I made it so it should look fine with any number of logos (1-5), so I will leave it as is until we see

**Comment by jessiofhall** [comment]
Great! Has this update been deployed to production?
